src/tests/PokemonsContainer.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import '@testing-library/jest-dom/extend-expect';
+import PokemonsContainer from '../components/PokemonsContainer';
+import store from '../redux/configureStore';
+
+describe('pokemons container component', () => {
+  test('matches PokemonsContainer component snapshot', () => {
+    const tree = renderer.create(
+      <Provider store={store}>
```

Comment:
- [ ] Some of your test cases are yet failing. Kindly ensure to fix the same.

**Failing test case:**
![image](https://user-images.githubusercontent.com/12745474/142707508-12125154-6c37-4786-9d72-7b779b5cb6cb.png)